### PR TITLE
FAQ improvements

### PR DESCRIPTION
**Pull request by rollerozxa**
Did a quick sweep over the FAQ page.

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -257,15 +220,15 @@ Minetest Game uses pre-1.8 Minecraft skins (64x32). These support regular body t
 
 If you play Minetest Game, you can build a [bed](https://wiki.luanti.org/Bed "Bed") and sleep at night. On your next life, you will spawn on the bed.
 
-Otherwise, you set your spawn point for _all_ worlds using `static_spawnpoint = (x,y,z)` in your [minetest.conf](https://wiki.luanti.org/Minetest.conf "Minetest.conf") file.
+Otherwise, you set your spawn point for _all_ worlds using `static_spawnpoint = (x,y,z)` in your `minetest.conf` file.
```
Comment (wsor4035) [inline content/faq.md:223]: link to example.minetest.conf in engine repo?

**Review by wsor4035 — CHANGES_REQUESTED** [review]
one thought, feel free to say no

File: content/faq.md (hunk 1 above)
Comment (rollerozxa) [reply content/faq.md]: Good idea, yes.

**Review by wsor4035 — APPROVED** [review]
lgtm

**Pull request merged**